### ISSUE TITLE
question on PSNR

### ISSUE THREAD

**Issue by lu123456789** [ingra14m/Deformable-3D-Gaussians#79]
Hello, thank you for your excellent work.
I experiment on lego using the validation set of Lego as the test set and run by "python train.py -s /lego/ -m output/lego --eval --is_blender --is_6dof". 
But I got the PSNR of 29.7, which is lower than the 33.07 in the paper. What could be the cause of this?
![image](https://github.com/user-attachments/assets/bf313902-65d3-4dfc-8c58-b5ff8c0efd64)

**Comment by ingra14m** [comment]
Hi, I also use the first image in the validation set as the training data. Please refer to [data](https://github.com/ingra14m/Deformable-3D-Gaussians/releases/tag/v0.1-pre-released) used in Deformable-GS for more information.

**Comment by lu123456789** [comment]
Thank you.
